NewForce-at-Mountwest/bangazon-api-bucatini
# Bf product controller

# Description

I added a class for Product as well as a controller and a test to check the ProductController.


## Related Ticket(s)
Ticket 2

## Steps to Test
After you pull down branch bf-product-controller, you should be able to begin test from ProductTest.cs which will check for proper GET all products, GET a product, PUT and DELETE calls. All six test should pass. Delete functionality is only soft deleting by changing Archived key to true instead of permanently deleting from DB. Only permanent delete are those Products created in the processing of tests.


# Checklist:
